### PR TITLE
enhancement: Reorganized navbar #70

### LINKED ISSUE #70: [Enhancement]: Reorganise navbar
### My idea

1. Remove navbar from `base.html`
2. Create new template for websites with normal navbar (It should have only link to login page and title)
3. Create separare navbar in `panel.html` with profile picture etc.
4. Make all links in dashboard prefixed with `panel`
5. Rename `gwardia/index.html` to `gwardia/mettings.html` (also change URL)

### PR DESCRIPTION
**Pull request by Sanielia**

### CODE REVIEW

File: core/templates/core/panel.html [hunk 1]
```diff
@@ -5,6 +5,29 @@
 {% load list_events %}
 
 {% block content %}
+    <nav class="flex flex-nowrap justify-between w-full p-1 bg-gray-950 border-b border-gray-600 text-gray-300">
+    <div class="flex flex-nowrap items-center gap-4">
```
Comment (Norbiros) [inline core/templates/core/panel.html]: Indent!

File: gwardia/urls.py [hunk 2]
```diff
@@ -4,5 +4,6 @@
 
 app_name = "gwardia"
 urlpatterns = [
-    path("", views.index, name="index"),
+    path("", views.meetings, name="index"),
+    path("asdf", views.asdf, name="asdf"),
```
Comment (Norbiros) [inline gwardia/urls.py]: ?

File: gwardia/views.py [hunk 3]
```diff
@@ -22,7 +22,11 @@
 }
 
 
-def index(request):
+def asdf(a):
+    asdf(1)
```
Comment (Norbiros) [inline gwardia/views.py]: Why?

File: core/templates/core/navbar.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<nav class="flex flex-nowrap justify-between w-full p-1 bg-gray-950 border-b border-gray-600 text-gray-300">
```
Comment (Norbiros) [inline core/templates/core/navbar.html]: Why not in `templatetahs`?

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: gwardia/views.py (hunk 3 above)
Comment (Sanielia) [reply gwardia/views.py]: eeeeeeeeeeeeeeee

File: gwardia/urls.py (hunk 2 above)
Comment (Sanielia) [reply gwardia/urls.py]: eeeeeeeeee

File: core/templates/core/panel.html (hunk 1 above)
Comment (Sanielia) [reply core/templates/core/panel.html]: okok

File: core/templates/core/navbar.html (hunk 4 above)
Comment (Sanielia) [reply core/templates/core/navbar.html]: em I did it the same way with `list_events.html` and I thought that you should put template which is assigned to templatetag to templates

File: .gitignore [hunk 5]
```diff
@@ -75,7 +75,6 @@ celerybeat-schedule.*
 *.sage.py
 
 # Environments
-.env
```
Comment (Norbiros) [inline .gitignore:78]: ???

File: gwardia/views.py [hunk 6]
```diff
@@ -5,7 +5,7 @@
 
 SIDEBAR_LINKS = {
     "Gwardia": [
-        {"name": "Spotkania", "url": "/gwardia", "icon": "calendar-days"},
+        {"name": "Spotkania", "url": "/panel/gwardia", "icon": "calendar-days"},
```
Comment (Norbiros) [inline gwardia/views.py]:
```suggestion
        {"name": "Spotkania", "url": "/panel/gwardia/meetings", "icon": "calendar-days"},
```

File: core/views.py [hunk 7]
```diff
@@ -2,7 +2,15 @@
 
 
 def index(request):
-    return render(request, "core/index.html")
+    context = {
+        "navbar_links": [
+            {
+                "name": "Gwardia",
```
Comment (Norbiros) [inline core/views.py]:
```suggestion
                "name": "Panel",
```

File: core/views.py [hunk 8]
```diff
@@ -2,7 +2,15 @@
...
+                "url": "panel/gwardia/",
```
Comment (Norbiros) [inline core/views.py]:
```suggestion
                "url": "panel/",
```

File: core/templates/core/panel.html [hunk 9]
```diff
@@ -5,6 +5,29 @@
...
+    <nav class="flex flex-nowrap justify-between w-full p-1 bg-gray-950 border-b border-gray-600 text-gray-300">
+        <div class="flex flex-nowrap items-center gap-4">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center transition-colors duration-300 ml-2 md:ml-5 hover:text-blue-100" target="_blank">
+                <span class="fa-brands fa-github fa-2x"></span>
+            </a>
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="text-md md:text-xl font-bold">GwardiaHub</a>
+        </div>
+        <ul class="flex justify-between items-center gap-4 m-2">
+            {% for link in navbar_links %}
+            <li>
```
Comment (Norbiros) [inline core/templates/core/panel.html]: Indent

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (Sanielia) [reply .gitignore:78]: .env is already on line 8 in `.gitignore`

Comment (Sanielia) [reply .gitignore:78]: It's duplicazted (there is also `.env` on line 8)

**Review by Norbiros — DISMISSED** [review]

File: core/templates/core/index.html [hunk 10]
```diff
@@ -24,16 +26,11 @@
 {% endblock %}
 
 {% block content %}
-    <canvas id="starback-bg" class="fixed -z-10"></canvas>
+    {% navbar navbar_links %}
```
Comment (goteusz-maszyk) [inline core/templates/core/index.html]: remove navbar from "/"

File: core/templates/core/panel.html [hunk 11]
```diff
@@ -5,14 +5,33 @@
 {% load list_events %}
 
 {% block content %}
+    <nav class="flex flex-nowrap justify-between w-full p-1 bg-gray-950 border-b border-gray-600 text-gray-300">
+        <div class="flex flex-nowrap items-center gap-4">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center transition-colors duration-300 ml-2 md:ml-5 hover:text-blue-100" target="_blank">
+                <span class="fa-brands fa-github fa-2x"></span>
+            </a>
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="text-md md:text-xl font-bold">GwardiaHub</a>
```
Comment (goteusz-maszyk) [inline core/templates/core/panel.html:13]:
```suggestion
            <a href="/" class="text-md md:text-xl font-bold">GwardiaHub</a>
```

File: core/templates/core/panel.html [hunk 12]
```diff
@@ -5,14 +5,33 @@
...
+    </nav>
     <div class="fixed bg-gray-900 opacity-70 h-full w-full hidden" id="dark-overlay"></div>
     <div class="flex flex-row h-full overflow-auto">
         <div id="sidebar" class="flex flex-col gap-y-3 fixed md:relative top-0 -translate-x-72 md:translate-x-0
             duration-300 min-w-72 w-72 h-full bg-gray-800 overflow-auto border-r border-gray-600">
             <a href="/" class="mx-4 pt-4 text-lg font-bold hover:underline">Strona Główna</a>
             {% for namespace, links in sidebar_links.items %}
-                <div class="flex flex-col divide-y divide-y-reverse divide-gray-600 gap-4">
-                    <h3 class="mx-4 py-2 text-lg font-bold">{{ namespace }}</h3>
+                <div class="flex flex-col divide-y divide-y-reverse divide-gray-600 gap-2">
```
Comment (goteusz-maszyk) [inline core/templates/core/panel.html]: I feel like my brain is compressed when looking at this

```suggestion
                <div class="flex flex-col divide-y divide-y-reverse divide-gray-600 gap-3">
```

File: core/templates/core/profile.html [hunk 13]
```diff
@@ -1,14 +1,14 @@
...
 
-{% block content %}
-    <div class="mt-32 flex flex-col items-center">
+{% block panel_content %}
+    <div class="mt-32 flex flex-col items-center w-full">
         <div class="flex gap-5 md:w-1/2 w-3/4 pb-4 border-b-2">
-            <img src="{% if user.avatar %}{{ user.avatar }}{% else %}{% static "images/placeholder_profile_pic.png" %}{% endif %}" alt="Profile picture" class="lg:w-[12.5%] lg:h-[12.5%] md:w-1/6 md:h-1/6 w-1/4 h-1/4 rounded-full">
+            <img src="{% if user.avatar %}{{ user.avatar }}{% else %}{% static "images/placeholder_profile_pic.png" %}{% endif %}" alt="Profile picture" class="h-20 w-20 object-cover rounded-full">
             <div class="flex flex-col">
                 <h1 class="text-4xl text-bold">{{ user.name }}</h1>
```
Comment (goteusz-maszyk) [inline core/templates/core/profile.html:14]: make it mobile-friendly

File: core/urls.py [hunk 14]
```diff
@@ -5,5 +5,5 @@
 app_name = "core"
 urlpatterns = [
     path("", views.index, name="index"),
-    path("profile/", views.profile, name="profile"),
+    path("panel/profile/<str:name>", views.profile, name="profile"),
```
Comment (goteusz-maszyk) [inline core/urls.py:8]: why?

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

File: core/urls.py (hunk 14 above)
Comment (Norbiros) [reply core/urls.py:8]: 1. It should be in panel
2. What if you want to view profile of other person?

Comment (goteusz-maszyk) [reply core/urls.py:8]: Oh so it's intended to be able to view other's profiles

File: core/templates/core/panel.html (hunk 11 above)
Comment (Norbiros) [reply core/templates/core/panel.html:13]: We have link to `/` in sidebar. Here we have GitHub logo so link to gh can stay

**Review by Norbiros — COMMENTED** [review]
LGTM, but I changed code in this PR so my review is kinda invalid :3

File: core/templates/core/index.html [hunk 15]
```diff
@@ -24,16 +24,15 @@
...
 
-{% block navbar_links %}
-    <li>
-        <a href="{% url 'gwardia:index' %}" class="hover:underline">Gwardia</a>
-    </li>
+        {# ToDo: Change this link /login if user is not logged in #}
+        <a class="mt-10 p-5 cursor-pointer ease-in-out duration-500 border-4 font-bold text-xl rounded-2xl bg-gradient-to-r from-[#4f16d9] to-[#9400ff] hover:shadow-[0_0_20px_10px_#4f16d9] hover:scale-110" href="{% url "gwardia:meetings" %}">
+            Otwórz panel
+        </a>
+    </div>
```
Comment (goteusz-maszyk) [inline core/templates/core/index.html:37]: this button is really ugly

File: core/urls.py (hunk 14 above)
Comment (goteusz-maszyk) [inline core/urls.py:8]: I think profile viewer is not part of a panel. If it is, then what isn't?

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

File: core/urls.py (hunk 14 above)
Comment (Norbiros) [reply core/urls.py:8]: Panel = navigation. So everything for logged users is inside panel

File: core/templates/core/index.html (hunk 15 above)
Comment (Norbiros) [reply core/templates/core/index.html:37]: nope. it's okay and it matches one in #63

Comment (WitaLisek) [reply core/templates/core/index.html:37]: I'm making standarts here :)

**Review by hexxcatee — DISMISSED** [review]
idk lgtm :333

**Review by goteusz-maszyk — DISMISSED** [review]

**Review by Norbiros — COMMENTED** [review]
LGTM, but I changed code in this PR so my review is kinda invalid :3`

**Review by hexxcatee — APPROVED** [review]
✅✅✅

**Review by tapczanek1321 — APPROVED** [review]
supersupersuper

**Pull request merged**